ml-agents/mlagents/trainers/ppo/optimizer_torch.py

```diff
@@ -168,7 +168,7 @@ def update(self, batch: AgentBuffer, num_sequences: int) -> Dict[str, float]:
         value_loss = self.ppo_value_loss(
             values, old_values, returns, decay_eps, loss_masks
         )
-        policy_loss = self.ppo_policy_loss(
+        policy_loss = 2 * self.ppo_policy_loss(
```

Comment:
This is the reason why the PR is a WIP. The Policy Loss is 2x lower than in TF, and I can't figure out why. Adding the 2x factor here makes PPO train better.